### DISCUSSION TITLE
best practice for use of (local) font awesome icons?

### DISCUSSION THREAD

**Discussion by larsneo** [statamic/cms#4600] [Q&A] [answered]
i'm looking for a simple way to include a couple of font-awesome icons (full package hosted locally in /node_modules/@fortawesome) in some of my templates. the existing v3-addon seems to be overhead and even more uses external javascript, so that's no option due to GDPR concerns. is there a simple way to accomplish this for a complete newbie like me?

**Comment by jackmcdade** [comment, marked as answer]
There's a whole guide on their site on how to do that – no addon necessary: https://fontawesome.com/v5.15/how-to-use/on-the-web/setup/hosting-font-awesome-yourself
